src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -110,7 +110,7 @@ void EquationInputArea::EquationTextBox_KeyGraphFeaturesButtonClicked(Object ^ s
     auto tb = static_cast<EquationTextBox ^>(sender);
     auto eq = static_cast<EquationViewModel ^>(tb->DataContext);
     EquationVM = eq;
```

Comment:
Do we still need to assign eq to EquationVM or can we just remove that line and have the event send eq?

Reply:
Yes we are still binding to EquationVM in xaml